ConsoleUI/ModListScreen.cs

```diff
@@ -303,6 +303,15 @@ public ModListScreen(GameInstanceManager mgr, bool dbg)
             mainMenu = new ConsolePopupMenu(opts);
         }
 
+        public override void Run(ConsoleTheme theme, Action<ConsoleTheme> process = null)
+        {
+            if (!((Registry) registry)?.HasAnyAvailable() ?? false)
+            {
+                UpdateRegistry(theme);
```

Comment:
Could we do this in `GetAllMods` instead? Overriding `Run` just to add some extra initialization makes me slightly uncomfortable, I fear we'll end up adding more and more logic here when there might be better places for it.

Reply:
Ah, that looks like a good place! Did find that one when I looked through the files :smile: